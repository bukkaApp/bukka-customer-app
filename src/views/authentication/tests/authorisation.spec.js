import React from 'react';
import Authorisation from 'src/components/Authentication';

describe('authorisation component', () => {
  const props = {
    title: 'Sign Up',
    handleChange: jest.fn(),
    handleSubmit: jest.fn(),
    domStructure: [
      {
        name: 'firstName',
        type: 'text',
        placeholder: 'First Name',
        id: 'firstName'
      }
    ],
    isFormCompleted: true,
    type: 'signin'
  };
  const wrapper = shallow(<Authorisation {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
