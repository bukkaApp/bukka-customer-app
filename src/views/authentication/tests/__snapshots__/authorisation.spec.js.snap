// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authorisation component renders properly 1`] = `ShallowWrapper {}`;
